E2E/tests/gameComplete.test.ts

```diff
@@ -0,0 +1,29 @@
+Feature('game complete');
+
+Before(async ({ I, login, username }) => {
+  I.amOnPage('/');
+  login.login(username.user1);
+});
+
+Scenario('Game completes automatically after last voter votes', async ({ I, login, username, roles, game, chessPiece, chessTile }) => {
```

Comment:
{ I, login, username, roles, game, chessPiece, chessTile } -I think this part could be described once at the beginning of this file , so will be no need to repeat it in other scenarios:
const {  I, login, username, roles, game, chessPiece, chessTile } = inject();